homeassistant/components/utility_meter/const.py

```diff
@@ -49,5 +48,4 @@
 
 SERVICE_RESET = "reset"
 SERVICE_SELECT_TARIFF = "select_tariff"
-SERVICE_SELECT_NEXT_TARIFF = "next_tariff"
```

Comment:
I think SERVICE_SELECT_TARIFF constant can also be dropped

Reply:
well spotted